utility/add_id_seq.sql

```diff
@@ -0,0 +1,388 @@
+--
+-- This script is to fix various issues in legacy database.
+-- It should be run on the database before running any
+-- nature migrations.
+--
+
+--
+-- The database seems to have multiple schemas, and the tables
+-- are created in public schema.
+--
+SET search_path=public;
+
+--
```

Comment:
Apparently we don't have the new version database with the `hav_koodi` column yet, so let's add it here if it's not found in the legacy database:

`ALTER TABLE lajihavainto ADD COLUMN IF NOT EXISTS hav_koodi VARCHAR(100);
ALTER TABLE lajirekisteri ALTER COLUMN koodi TYPE VARCHAR(100);`

Please check that this works :)